src/einsteinpy/symbolic/__init__.py

```diff
@@ -4,6 +4,6 @@
 from .ricci import RicciScalar, RicciTensor
 from .riemann import RiemannCurvatureTensor
 from .stress_energy_momentum import StressEnergyMomentumTensor
-from .tensor import Tensor
+from .tensor import GRTensor, Tensor
```

Comment:
Instead of GRTensor, name it BaseTensor :D 
That would be more appropriate!

Reply:
> A little consideration. Everything looks fine. Any hopes of getting green tick on this?'
Codeclimate can't be fixed. There are too many exceptions to throw !